lib/runners/go.js

```diff
@@ -53,6 +53,8 @@ func Test${name}(t *testing.T) {
         // exit status 1
         // FAIL	{dirname of test file}	{seconds}s
         sanitizeStdOut: function(stdout) {
+            // special case when build failed
+            if (stdout.includes('[build failed]')) return '<ERROR::>Build Failed\n';
```

Comment:
After '[build failed]', stdout contains the same information as stderr (last 3 lines are red).

```
FAIL	codewarrior/kata [build failed]
# codewarrior/kata
./kata.go:3: a + b evaluated but not used
./kata.go:4: missing return at end of function

      ✓ should have a passing bug fixes example (808ms)
# codewarrior/kata
./kata.go:3: a + b evaluated but not used
./kata.go:4: missing return at end of function
```
becomes
```
<ERROR::>Build Failed
# codewarrior/kata
./kata.go:3: a + b evaluated but not used
./kata.go:4: missing return at end of function
      1) should have a passing bug fixes example
```